scripts/dev_setup.sh

```diff
@@ -932,14 +979,10 @@ if [[ "$INSTALL_BUILD_TOOLS" == "true" ]]; then
   install_cargo_sort
   install_cargo_nextest
   install_grcov
-  install_postgres
   install_pkg git "$PACKAGE_MANAGER"
   install_lcov "$PACKAGE_MANAGER"
-  install_nodejs "$PACKAGE_MANAGER"
-  install_pnpm "$PACKAGE_MANAGER"
   install_pkg unzip "$PACKAGE_MANAGER"
   install_protoc
-  install_solidity
```

Comment:
postgres was used for the old indexer
nodejs was likely used for some old legacy
pnpm is only required for sdks
solidity is currently required for testing move, afaiu -- so it is probably required -- question I have is that something we intend to continue to support or should we consider sun setting it, don't know if we'll run into any friction in future development...

Reply:
I added back node.js, pnpm, and solidity in the CI setup scripts so that it passes tests.